ra_aid/__main__.py

```
@@ -650,9 +765,181 @@ def log_level_type(value):
+def handle_extract_last_plan(args):
+    from ra_aid.scripts.extract_last_plan import main as extract_last_plan_main
+    extract_last_plan_main(project_state_dir=args.project_state_dir)
+    sys.exit(0)
+
+def handle_extract_last_research_notes(args):
+    from ra_aid.scripts.extract_last_research_notes import main as extract_last_research_notes_main
+    extract_last_research_notes_main(project_state_dir=args.project_state_dir)
+    sys.exit(0)
+
+def handle_generate_openapi(args):
+    from ra_aid.scripts.generate_openapi import main as generate_openapi_main
```

Pylint (3.3.7)

[error] No name 'main' in module 'ra_aid.scripts.generate_openapi'

(E0611)